### PR TITLE
labelImg: 1.8.3 -> 1.8.6

### PR DESCRIPTION
**Pull request by TheDelus**
###### Description of changes

Version bump to 1.8.6
Patches an issue with newer python versions that is already patched upstream but not "released" yet
See: https://github.com/heartexlabs/labelImg/issues/838

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Cryolitia** [comment]
labelImg crashes when I click open folder

```
$ nix shell github:TheDelus/nixpkgs/bump-and-fix-labelImg#labelImg

$ labelImg

Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
/nix/store/w08svjsqrj7zncvzkd7fwa82nazwmsk1-dconf-0.40.0-lib/lib/gio/modules/libdconfsettings.so: undefined symbol: g_assertion_message_cmpint
Failed to load module: /nix/store/w08svjsqrj7zncvzkd7fwa82nazwmsk1-dconf-0.40.0-lib/lib/gio/modules/libdconfsettings.so
/nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/zx48rsavh81fz3winyz2cgbqmnmvdz6f-gvfs-1.52.1/lib/gio/modules/libgvfsdbus.so)
Failed to load module: /nix/store/zx48rsavh81fz3winyz2cgbqmnmvdz6f-gvfs-1.52.1/lib/gio/modules/libgvfsdbus.so
QApplication: invalid style override 'Nordic-Polar' passed, ignoring it.
	Available styles: Windows, Fusion

(python3.10:20624): Gtk-WARNING **: 23:30:57.207: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/sqxcdwx3vnyn0qyw777mzjl786q5a0ls-ibus-1.5.28/lib/libibus-1.0.so.5)

(python3.10:20624): Gtk-WARNING **: 23:30:57.207: Loading IM context type 'ibus' failed

(python3.10:20624): Gtk-WARNING **: 23:30:57.208: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/sqxcdwx3vnyn0qyw777mzjl786q5a0ls-ibus-1.5.28/lib/libibus-1.0.so.5)

(python3.10:20624): Gtk-WARNING **: 23:30:57.208: Loading IM context type 'ibus' failed

(python3.10:20624): Gtk-WARNING **: 23:30:57.208: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/sqxcdwx3vnyn0qyw777mzjl786q5a0ls-ibus-1.5.28/lib/libibus-1.0.so.5)

(python3.10:20624): Gtk-WARNING **: 23:30:57.208: Loading IM context type 'ibus' failed

(python3.10:20624): Gtk-WARNING **: 23:30:57.209: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/sqxcdwx3vnyn0qyw777mzjl786q5a0ls-ibus-1.5.28/lib/libibus-1.0.so.5)

(python3.10:20624): Gtk-WARNING **: 23:30:57.209: Loading IM context type 'ibus' failed

(python3.10:20624): Gtk-WARNING **: 23:30:57.210: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/sqxcdwx3vnyn0qyw777mzjl786q5a0ls-ibus-1.5.28/lib/libibus-1.0.so.5)

(python3.10:20624): Gtk-WARNING **: 23:30:57.210: Loading IM context type 'ibus' failed
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
/nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/zx48rsavh81fz3winyz2cgbqmnmvdz6f-gvfs-1.52.1/lib/gio/modules/libgvfsdbus.so)
Failed to load module: /nix/store/zx48rsavh81fz3winyz2cgbqmnmvdz6f-gvfs-1.52.1/lib/gio/modules/libgvfsdbus.so

(python3.10:20624): Gtk-WARNING **: 23:30:59.061: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /home/cryolitia/.nix-profile/share/icons/Adwaita/scalable/status/image-missing.svg: Unable to load image-loading module: /nix/store/z2bfc9frkn6izdps38v0kmsyfxvkmm25-librsvg-2.57.0/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libm.so.6: version `GLIBC_2.38' not found (required by /nix/store/z2bfc9frkn6izdps38v0kmsyfxvkmm25-librsvg-2.57.0/lib/librsvg-2.so.2) (gdk-pixbuf-error-quark, 5)
Bail out! Gtk:ERROR:../gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /home/cryolitia/.nix-profile/share/icons/Adwaita/scalable/status/image-missing.svg: Unable to load image-loading module: /nix/store/z2bfc9frkn6izdps38v0kmsyfxvkmm25-librsvg-2.57.0/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /nix/store/1nyg1fvhpz8bx3vn3r9f18zhra2rpbx9-glibc-2.37-8/lib/libm.so.6: version `GLIBC_2.38' not found (required by /nix/store/z2bfc9frkn6izdps38v0kmsyfxvkmm25-librsvg-2.57.0/lib/librsvg-2.so.2) (gdk-pixbuf-error-quark, 5)
zsh: abort (core dumped)  labelImg

```

**Comment by Cryolitia** [comment]
seems like https://github.com/NixOS/nixpkgs/issues/271549

File: pkgs/applications/science/machine-learning/labelimg/default.nix [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
   python3Packages.buildPythonApplication rec {
     pname = "labelImg";
-    version = "1.8.3";
+    version = "1.8.6";
     src = fetchFromGitHub {
       owner = "tzutalin";
       repo = "labelImg";
       rev = "v${version}";
-      sha256 = "07v106fzlmxrbag4xm06m4mx9m0gckb27vpwsn7sap1bbgc1pap5";
+      sha256 = "sha256-RJxCtiDOePajlrjy9cpKETSKsWlH/Dlu1iFMj2aO4XU=";
```
Comment (Cryolitia) [inline pkgs/applications/science/machine-learning/labelimg/default.nix:14]:
```suggestion
      hash = "sha256-RJxCtiDOePajlrjy9cpKETSKsWlH/Dlu1iFMj2aO4XU=";
```

**Review by Cryolitia — CHANGES_REQUESTED** [review]

**Comment by Cryolitia** [comment]
works with https://github.com/Cryolitia/nixpkgs/tree/bump-and-fix-labelImg

**Comment by Cryolitia** [comment]
I have created a new PR https://github.com/NixOS/nixpkgs/pull/300266 due to your long time AFK. I'm very sorry if this offends you and I would close my PR if you come back to work.

**Comment by wegank** [comment]
Closed by #300266, where your authorship has been kept.

**Pull request closed without merging**